tests/test_pybats.py

```diff
@@ -329,12 +329,32 @@ def setUp(self):
     knownImfRho  = [5., 15.]
     knownImfTemp = [.80E+05, 1.20E+05]
     knownImfIono = [4.99, 0.01]
+    knownSubMilli= dt.datetime(2017, 9, 6, 16, 42, 36, 999000)
 
     def tearDown(self):
```

Comment:
Can confirm that tearDown exists and works properly on my machine.  @drsteve, can you confirm that tearDown is working for you?